crates/fluvio-version-manager/src/command/itself/mod.rs

```diff
@@ -0,0 +1,48 @@
+//! FVM Management Commands
+
+pub mod install;
+pub mod uninstall;
+
+use color_eyre::eyre::Result;
+use clap::Parser;
+
+use crate::GlobalOptions;
+use crate::common::notify::Notify;
+
+use self::install::InstallOpt;
+use self::uninstall::UninstallOpt;
+
+#[derive(Debug, Parser)]
+pub enum ItselfCommand {
+    /// Install `fvm` and setup the workspace
+    Install(InstallOpt),
+    /// Uninstall `fvm` and removes the workspace
+    Uninstall(UninstallOpt),
+}
+
+/// The `install` command is responsible of installing the desired Package Set
+#[derive(Debug, Parser)]
+pub struct ItselfOpt {
+    #[command(flatten)]
+    global_opts: GlobalOptions,
+    /// Subcommand to execute
+    #[clap(subcommand)]
+    command: ItselfCommand,
+}
+
+impl ItselfOpt {
+    pub async fn process(&self) -> Result<()> {
+        match &self.command {
+            ItselfCommand::Install(cmd) => cmd.process().await?,
+            ItselfCommand::Uninstall(cmd) => cmd.process().await?,
+        }
+
+        Ok(())
+    }
+}
+
+impl Notify for ItselfOpt {
```

Comment:
why this is repeated here?

Reply:
Because this provides `ItselfOpt` context to determine if output should be printed.
I plan to do a derive macro later so we reduce boilerplate.

This makes the `GlobalConfig` `is_quiet` prop acknowledged when printing to stdout.

Reply:
Do we need to this now?  Just add print when needed.  Otherwise you can just add to logging.

Reply:
Its for user feedback. `Notify` provides  methods to keep it standardized on all commands

Reply:
Another advantage is the fact that is config aware, every command has access to the top-level `GlobalConfig`, `Notify` takes advantage of this to use as global flag for no output which is helpful in CI Tests if we want to focus on certain output.

Looking for other impls in `fluvio-cli`, I found uses on `println!`

https://github.com/infinyon/fluvio/blob/master/crates/fluvio-cli/src/install/plugins.rs#L115-L126

The disadvantage is that the user doesn't have an option to suppress this, other than piping output somewhere else.

Reply:
Let's get something working and then improve rather than trying to do everything once.  If this is absolutely needed then let's use(improve) the `terminal` interface which is in common.   No need to have two different ways of doing things